e2e/process-services-cloud/form-field/visibility-condition.e2e.ts

```diff
@@ -248,3 +253,155 @@ describe('Visibility conditions - cloud', () => {
+        testUser = await identityService.createIdentityUserWithRole(apiService, [identityService.ROLES.APS_USER]);
+        groupInfo = await groupIdentityService.getGroupInfoByGroupName('hr');
+        await identityService.addUserToGroup(testUser.idIdentityService, groupInfo.id);
+
+        await settingsPage.setProviderBpmSso(
+            browser.params.config.bpmHost,
+            browser.params.config.oauth2.host,
+            browser.params.config.identityHost);
+        await loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+
+    });
+
+    afterAll(async () => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+        await identityService.deleteIdentityUser(testUser.idIdentityService);
+
+    });
+
+    beforeEach(async () => {
+        await navigationBarPage.navigateToProcessServicesCloudPage();
+        await appListCloudComponent.checkApsContainer();
+        await appListCloudComponent.goToApp(simpleApp);
+        await tasksCloudDemoPage.taskListCloudComponent().checkTaskListIsLoaded();
+    });
+
+    it('[C312400] Should validate visibility condition for required number widgets', async () => {
+        await tasksCloudDemoPage.openNewTaskForm();
+        await startTask.checkFormIsDisplayed();
+        await startTask.addName(standaloneTaskName);
+        await startTask.selectFormDefinition('requirednumbervisibility');
+        await startTask.clickStartButton();
+        await tasksCloudDemoPage.taskListCloudComponent().selectRow(standaloneTaskName);
+
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number1');
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 5);
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 123);
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+    });
+
+    it('[C315169] Should be able to start a process with visibility condition for number widgets', async () => {
+        const processModelWithSe = 'numbervisibilityprocess';
+        const numberVisibilityTask = 'number_visibility_task';
+        processName = StringUtil.generateRandomString(5);
+
+        await processCloudDemoPage.openNewProcessForm();
+        await startProcessPage.clearField(startProcessPage.processNameInput);
+        await startProcessPage.selectFromProcessDropdown(processModelWithSe);
```

Comment:
The constant is never used outside this call
```suggestion
        await startProcessPage.selectFromProcessDropdown('numbervisibilityprocess');
```